summary-relationships/src/index-executes.ts

```diff
@@ -2,6 +2,7 @@ import { getClient } from "./get-client";
 import { buildPayload } from "./build-payload";
 import { waitForJobFinalization } from "./wait-for-job";
 import { WorkloadAccessExecute } from "./workload-access-execute-query";
+const uniqBy = require("lodash.uniqby");
```

Comment:
Can we import using `import { uniqBy } from "lodash"`? Not sure how that library exports stuff, but it would be nice for consistency sake.

Reply:
👍